cpp/include/cudf/io/detail/orc.hpp

```diff
@@ -124,6 +124,11 @@ class writer {
    * @brief Finishes the chunked/streamed write process.
    */
   void close();
+
+  /**
+   * @brief Skip writing the footer and close the writer.
+   */
+  void skip_close();
```

Comment:
I can't think of a better name for this function.  I fear the name implies that it's the `close()` that's skipped.
`skip_footer_write_on_close()` is too long.  

If we can't think of a succinct name, maybe we should keep `skip_close()`.

Reply:
omg, my updated code for this function got lost between commits.

I think `skip_close` is a pretty accurate name, given that we raise the `_closed` flag in `skip_close` and `close()` returns immediately if that flag is raised. 

I'll update the docs above, sorry for the confusion.